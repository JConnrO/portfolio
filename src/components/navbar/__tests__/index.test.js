import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect'
import Navbar from '..';

afterEach(cleanup);

describe('Navbar Component', () => {
    // Baseline Render Test
    test('renders', () => {
        render(<Navbar />)
    })
    //Snapshot Test
    test('snapshot', () => {
        const { asFragment } = render(<Navbar/>)
        expect(asFragment()).toMatchSnapshot();
    })
})
