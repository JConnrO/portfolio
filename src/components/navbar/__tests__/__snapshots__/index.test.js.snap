// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component snapshot 1`] = `
<DocumentFragment>
  <nav>
    Hello World
  </nav>
</DocumentFragment>
`;
